# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1381529309 -7200
#      Sat Oct 12 00:08:29 2013 +0200
# Node ID b30bb9bb05c8a88d2bd0a1fc831a29d7a9bba7fa
# Parent  230ffaf80ac9b64accf5066846904fbc38d37bc5
change some editor shortcuts

* file-editor.cc(set_shortcuts): change shortcuts for run, comment and uncomment

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -1244,34 +1244,34 @@ file_editor::pasteClipboard ()
   }
 }
 
 void
 file_editor::set_shortcuts (bool set)
 {
   if (set)
     {
-      _comment_selection_action->setShortcut (Qt::ControlModifier + Qt::Key_7);
-      _uncomment_selection_action->setShortcut (Qt::ControlModifier + Qt::Key_8);
+      _comment_selection_action->setShortcut (Qt::ControlModifier + Qt::Key_R);
+      _uncomment_selection_action->setShortcut (Qt::ControlModifier + Qt::Key_T);
 
       _copy_action->setShortcut (QKeySequence::Copy);
       _cut_action->setShortcut (QKeySequence::Cut);
       _paste_action->setShortcut (QKeySequence::Paste);
       _context_help_action->setShortcut (QKeySequence::HelpContents);
       _context_doc_action->setShortcut (Qt::SHIFT + Qt::Key_F1);
 
       _find_action->setShortcut (QKeySequence::Find);
       _goto_line_action->setShortcut (Qt::ControlModifier+ Qt::Key_G);
 
       _next_bookmark_action->setShortcut (Qt::Key_F2);
       _previous_bookmark_action->setShortcut (Qt::SHIFT + Qt::Key_F2);
       _toggle_bookmark_action->setShortcut (Qt::Key_F7);
 
       _print_action->setShortcut (QKeySequence::Print);
-      _run_action->setShortcut (Qt::ControlModifier+ Qt::Key_R);
+      _run_action->setShortcut (Qt::Key_F5);
       _context_run_action->setShortcut (Qt::Key_F9);
 
       _context_edit_action->setShortcut (Qt::ControlModifier + Qt::Key_E);
       _save_action->setShortcut (QKeySequence::Save);
       _save_as_action->setShortcut (QKeySequence::SaveAs);
       _close_action->setShortcut (QKeySequence::Close);
 
       _redo_action->setShortcut (QKeySequence::Redo);
